utils/rust-setup.js

```diff
@@ -0,0 +1,109 @@
+const chalk = require('chalk');
+const reader = require("readline-sync");
+const os = require('os');
+const sh = require('shelljs');
+
+// script from https://rustup.rs/ with auto-accept flag "-y"
+const installRustupScript = "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y";
+const updatePath = 'source $HOME/.cargo/env';
+const addWasm32TargetScript = "rustup target add wasm32-unknown-unknown";
+// We should update PATH in the same script because every new Bash scripts are executed in a separate shell
+const updatePathAndAddWasm32TargetScript = updatePath + ' && ' + addWasm32TargetScript;
+
+const isUnix = os.platform() != 'win32';
```

Comment:
seems like more fair description is to have `isWindows  = os.platform() == 'win32'`